Gopkg.toml

```diff
@@ -56,11 +60,11 @@ ignored = ["github.com/fnproject/fn/cli",
 
 [[constraint]]
   name = "github.com/fsouza/go-dockerclient"
-  revision = "1b33efcb40bef3494af8bc6a1f73600f37a3c9d1"
+  revision = "v1.3.0"
```

Comment:
we need 1.3.1 for this bug fix https://github.com/fsouza/go-dockerclient/releases/tag/v1.3.1